### PR TITLE
Bump to v1.19.4

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL
@giuseppe @vrothberg @saschagrunert @TomSweeneyRedHat PTAL

File: contrib/rpm/buildah.spec [hunk 1]
```diff
@@ -100,6 +100,13 @@ make DESTDIR=%{buildroot} PREFIX=%{_prefix} install install.completions
 %{_datadir}/bash-completion/completions/*
 
 %changelog
+* Thu Jan 28, 2021 Dan Walsh <dwalsh@redhat.com> 1.19.4-1
```
Comment (TomSweeneyRedHat) [inline contrib/rpm/buildah.spec:103]: Wrong date

**Comment by TomSweeneyRedHat** [comment]
LGTM other than the day/date tweak.

**Review by saschagrunert — APPROVED** [review]
LGTM, beside @TomSweeneyRedHat's comment

**Review by vrothberg — COMMENTED** [review]
Can we wait for https://github.com/containers/buildah/pull/2984?

**Comment by rhatdan** [comment]
Rebased

File: contrib/rpm/buildah.spec [hunk 2]
```diff
@@ -100,6 +100,14 @@ make DESTDIR=%{buildroot} PREFIX=%{_prefix} install install.completions
 %{_datadir}/bash-completion/completions/*
 
 %changelog
+* Thu Feb 8, 2021 Dan Walsh <dwalsh@redhat.com> 1.19.4-1
```
Comment (TomSweeneyRedHat) [inline contrib/rpm/buildah.spec:103]: next pass, we'll need to touch this up to `Sat Feb 6, ...`

**Pull request merged**